### DISCUSSION TITLE
Dane Azure AD tenantów połączonych z moim Azure AD tenant.

### DISCUSSION THREAD

**Discussion by klucyszy** [AzureCommunityPL/group#230] [Pomoc]
Cześć, 

poszukuję informacji jak wydobyć dane tenantów połączonych z moim Azure AD tenantem. Nie mogę znaleźć odpowiedniego endpointu w Microsoft Graph API. Coś przeoczyłem?

Scenariusz. Azure AD Tenant 1 jest połączony z z Azure AD Tenant 2 oraz Azure AD Tenant 3.
Potrzebuję wydobyć informację listę tenantów czyli Azure AD Tenant 2 oraz Azure AD Tenant 3 oraz informacje o nich ( np. ich domena).

Pozdrawiam

**Comment by stic** [comment]
Hej,
Co masz na myśli pisząc 'połączony'? 
[Multi-tenant](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-convert-app-to-be-multi-tenant), [Azure AD B2B](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/b2b-fundamentals), partner z delegated admin privileges, a może coś zupełnie innego ;-)
